_posts/2022-11-19-sr2023-kickstart-and-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SR2023 Game Announced
+---
+
+Student Robotics 2023 was Kickstarted on Saturday 22nd September at the University of Southampton and on our [livestream][livestream]. It was great to see the excitement building at our first in-person Kickstart since SR2019.
+
+Our game this year, _Greed_, challenges teams to steal tokens from other scoring zones. With Bronze, Silver, and Gold tokens, each worth different amount of game points, teams must think carefully about the optimum strategy. To help teams locate tokens they have 2D barcode style markers attached that our computer vision library can detect. However each of the markers will identify themselves as exactly the same, so teams must use other sensors to detect which type of token is which. Bronze and Silver tokens are the same size but the Silver tokens weigh 300g more, and Gold Tokens are larger than Bronze and Silver. Teams are scored at the end of the match for collecting the most points. Full details, including the prizes available this year, are available in the [rulebook][rules].
+
+This year we have also introduced a new way for teams to earn league points, Challenges! There are three challenges which teams may optionally complete during the competition year in order. These challenges encourage teams to start work on their robots early and cover movement, sensors, and vision. The challenges may be approached in any order, and completing challenges before certain deadlines will earn the teams bonus league points. Full details of the challenges are available in the [rulebook][rules].
```

Comment:
Just a thought (not sure if good): do we want to link to the events which represent the deadlines? Feels like it might be hard to work it in, but might also be a good way to advertise that they're in the list of events.

Reply:
👍 not super elegant but added in [b97108d](https://github.com/srobo/website/pull/461/commits/b97108dd01e6a43f6b05eadf3c44bb8bda99f115)